app/javascript/components/GuitarInfo.js

```diff
@@ -7,28 +8,79 @@ import {
   Card,
   CardTitle,
   CardBody,
-  CardFooter
+  CardFooter,
+  Button,
+  Modal,
+  ModalVariant
 } from '@patternfly/react-core';
+import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
+import { getGuitarUrlWithId, OPEN, CLOSE } from './constants.js';
+
+const GuitarInfo = ({ id, name, url, price, description }) => {
+  const [isDeleteModalOpen, setDeleteModal] = useState(CLOSE);
+
+  const openDeleteModal = () => {
+    setDeleteModal(OPEN);
```

Comment:
I think now it is better, when we explicitly mention: 'OPEN' & 'CLOSE'
WYT @MariaAga ?